pkgs/applications/audio/bespokesynth/default.nix

```diff
@@ -0,0 +1,109 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, fetchzip
+, libjack2, alsa-lib, freetype, libX11, libXrandr, libXinerama, libXext, libXcursor
+, libGL, python3, ncurses, libusb
+, gtk3, webkitgtk, curl, xvfb-run, makeWrapper
+  # "Debug", or "Release"
+, buildType ? "Release"
+}:
+
+let
+  projucer = stdenv.mkDerivation rec {
```

Comment:
Maybe we should move projucer to top-level.

Reply:
Now that they switched to cmake after the release, we will be able to remove projucer for this package with the next update.